client/src/containers/CreateIssue/index.jsx

```diff
@@ -21,29 +24,26 @@ const assigneeOptions = [
   { key: 'dz', value: 'dz', text: 'Michael Algol' }
 ];
 
+const validationSchema = Yup.object().shape({
+  issueTitle: Yup.string()
+    .max(256, 'Maximum length - 255 characters')
```

Comment:
Знаю, что мелочь, но всё же - чего 256, а не 255?)

Reply:
Так, дивно. Але в доках саме так)
https://docs.google.com/document/d/1z4xOSCtbFVghIUkv_APUl9ZECaSCdwc1zzi8T8ubhT8/edit

Reply:
А, хорошо, просто поменяй тогда, чтоб текст соответствовал действительности
```suggestion
    .max(256, 'Maximum length - 256 characters')
```